docs/modules/ROOT/pages/type-definitions/schema-configuration/field-configuration.adoc

```diff
@@ -0,0 +1,436 @@
+
+For instance:
+
+[source, graphql, indent=0]
+----
+type Movie {
+    title: String!
+    description: String
+}
+
+type Actor {
+    name: String!
+    actedIn: [Movie!]!
+        @relationship(type: "ACTED_IN", direction: OUT)
+        @selectable(onRead: false, onAggregate: false)
+}
+----
+
+It will generate the type type `Actor`:
+
+[source, graphql, indent=0]
+----
+type Actor {
+  name: String!
+  actedInAggregate(where: MovieWhere, directed: Boolean = true): ActorMovieActedInAggregationSelection
+}
+----
+
+[NOTE]
+====
+Please note how the `actedInAggregate` is not affected by the argument `onAggregate`. To disable the generation of `actedInAggregate` see the `aggregate` argument of the directive xref::type-definitions/schema-configuration/field-configuration.adoc#_relationship[`@relationship`]
+====
+
+== `@settable`
+
+This directive sets the availability of the input field on creation and update mutations.
+It has two arguments:
+
+* **onCreate**: If disabled, this field will not be available on creation operations.
+* **onUpdate**: If disabled, this field will not be available on update operations.
+
+=== Definition
+
+[source, graphql, indent=0]
+----
+"""Instructs @neo4j/graphql to generate this input field for mutation."""
+directive @settable(onCreate: Boolean! = true, onUpdate: Boolean! = true) on FIELD_DEFINITION
+----
+
+==== Usage
+
+With this definition:
+
+[source, graphql, indent=0]
+----
+type Movie {
+    title: String!
+    description: String @settable(onCreate: true, onUpdate: false)
+}
+
+type Actor {
+    name: String!
+    actedIn: [Movie!]!
+        @relationship(type: "ACTED_IN", direction: OUT)
+}
+----
+
+The following input fields will be generated:
+
+[source, graphql, indent=0]
+----
+input MovieCreateInput {
+    description: String
+    title: String!
+}
+
+input MovieUpdateInput {
+    title: String
+}
+----
+
+This means the description can be set on creation, but it will not be available on update operations.
+
+=== `@settable` with Relationships
+
+This directive can be used along with relationship fields. 
+When an operation on a field is disabled this way, that relationship will not be available on top-level operations. 
+For example:
+
+[source, graphql, indent=0]
+----
+type Movie {
+    title: String!
+    description: String
+}
+
+type Actor {
+    name: String!
+    actedIn: [Movie!]!
+        @relationship(type: "ACTED_IN", direction: OUT)
+        @settable(onCreate: false, onUpdate: true)
+}
+----
+
+The following input fields will be generated:
+
+[source, graphql, indent=0]
+----
+input ActorCreateInput {
+  name: String!
+}
+
+input ActorUpdateInput {
+  name: String
+  actedIn: [ActorActedInUpdateFieldInput!]
+}
+----
+
+This means the `actedIn` can be updated on an update, but it will not be available on create operations.
```

Comment:
Is is possible to add a section describing the difference between `nestedOperations` `CREATE`/`UPDATE` and `@settable` `onCreate`/`onUpdate`?

i.e. dissabling the nested create prevents creating a movie node from the actor node (but it would still be possible to create an actor and connect it to an existing movie), whereas `onCreate` would mean that the actor cannot be created with an actedIn relationship at all

For nested updates it prevents updating movie fields from an actor (but is still possible to connect/disconnect/delete/etc movies), whereas `onUpdate` makes the relationships and related types immutable (if starting at the actor)